### ISSUE TITLE
Taiwan city_choices list is empty

### ISSUE THREAD

**Issue by errnesto** [mirumee/google-i18n-address#69]
When I get the validation rules for Taiwan, `city` is a required field but `city_choices` is a empty list:

```python
ValidationRules(country_code='TW', country_name='TAIWAN', address_format='%Z%n%S%C%n%A%n%O%n%N', address_latin_format='%N%n%O%n%A%n%C, %S %Z', allowed_fields={'street_address', 'name', 'postal_code', 'country_area', 'company_name', 'city'}, required_fields={'country_area', 'postal_code', 'city', 'street_address'}, upper_fields={'city'}, country_area_type='county', country_area_choices=[('台中市', 'Taichung City'), ('台中市', '台中市'), ('台北市', 'Taipei City'), ('台北市', '台北市'), ('台東縣', 'Taitung County'), ('台東縣', '台東縣'), ('台南市', 'Tainan City'), ('台南市', '台南市'), ('宜蘭縣', 'Yilan County'), ('宜蘭縣', '宜蘭縣'), ('花蓮縣', 'Hualien County'), ('花蓮縣', '花蓮縣'), ('金門縣', 'Kinmen County'), ('金門縣', '金門縣'), ('南投縣', 'Nantou County'), ('南投縣', '南投縣'), ('屏東縣', 'Pingtung County'), ('屏東縣', '屏東縣'), ('苗栗縣', 'Miaoli County'), ('苗栗縣', '苗栗縣'), ('桃園市', 'Taoyuan City'), ('桃園市', '桃園市'), ('高雄市', 'Kaohsiung City'), ('高雄市', '高雄市'), ('基隆市', 'Keelung City'), ('基隆市', '基隆市'), ('連江縣', 'Lienchiang County'), ('連江縣', '連江縣'), ('雲林縣', 'Yunlin County'), ('雲林縣', '雲林縣'), ('新北市', 'New Taipei City'), ('新北市', '新北市'), ('新竹市', 'Hsinchu City'), ('新竹市', '新竹市'), ('新竹縣', 'Hsinchu County'), ('新竹縣', '新竹縣'), ('嘉義市', 'Chiayi City'), ('嘉義市', '嘉義市'), ('嘉義縣', 'Chiayi County'), ('嘉義縣', '嘉義縣'), ('彰化縣', 'Changhua County'), ('彰化縣', '彰化縣'), ('澎湖縣', 'Penghu County'), ('澎湖縣', '澎湖縣')], city_type='city', city_choices=[], city_area_type='suburb', city_area_choices=[], postal_code_type='postal', postal_code_matchers=[re.compile('^\\d{3}(?:\\d{2,3})?$')], postal_code_examples=['104', '106', '10603', '40867'], postal_code_prefix='')
```

So I can not manage to validate a address:

```python
try:
     normalize_address({'city': 'Taichung City', 'country_area': 'Taichung City', 'postal_code': '401', 'street_address': 'hello', 'country_code': 'TW'})
except InvalidAddress as e:
     print(e.errors)
 
# {'city': 'invalid'}

```
```python
try:
     normalize_address({'city': '', 'country_area': 'Taichung City', 'postal_code': '401', 'street_address': 'hello', 'country_code': 'TW'})
except InvalidAddress as e:
     print(e.errors)
 
# {'city': 'required'}
```

**Comment by patrys** [comment]
Did you pass a country area? The city choice list will depend on the country area, and the city area choice list will depend on the city.

**Comment by errnesto** [comment]
Oh yes thank you! that solved it.

As an example this works:
```python
address = normalize_address({'city': 'Daya District', 'country_area': 'Taichung City', 'postal_code': '428', 'street_address': 'hello', 'country_code': 'TW'})
```

It is just a bit strange to me to enter the city as `country_area` and the district as `city` but then I don't know anything about addresses in Taiwan…

**Comment by patrys** [comment]
@errnesto The validation API also contains information on what individual fields represent in any given country, see `country_area_type`, `city_type`, and `city_area_type` on the ValidationRules object.

**Comment by errnesto** [comment]
Hm yes I saw that. In the Taiwan case  it still does not really behave like I would expect it… but I guess this has also to do with the underlying data. Or maybe just because I don’t understand the political division in counties or cities there… 

Anyway thanks a lot for your help and quick responses!
I did not know The rules change when I add a area to the request. That helped me a lot to resolve the issue that caused me to open this.
